webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -344,6 +347,7 @@ ManageManifestModal.defaultProps = {
   isManifestImported: false,
   canEditOrganizations: false,
   simpleContentAccess: false,
+  simpleContentAccessEligible: false,
```

Comment:
I clicked on my manifest modal and saw the switch in the state of "on, but disabled" for a second.  If SCA is already on, it wouldn't make much sense for it not to be available.

During that moment when the API request is in flight and we don't know the answer, should the switch be enabled by default rather than disabled?
Should this change according to whether SCA is already enabled?

Reply:
Anything can happen. It's entirely possible that the imported manifest could have SCA toggled on but at some later point the org in the portal disabled SCA toggling across the board. Let me think about the 2nd part of your question

Reply:
I'll need to test this:

- i have an imported manifest with SCA enabled
- my upstream org has SCA disabled globally
- am I still able to toggle to 'off' in Katello? (need to test and handle this case, if that works which I think it should)

Reply:
Updated to allow disabling SCA even if the upstream org is ineligible. That way users can switch back to entitlement mode if they wish, but they won't be able to toggle it back on.